orcavault/models/mart/centre/lims.sql

```diff
@@ -33,6 +33,15 @@ with effective_link_library_experiment as (
 
 ),
 
+effective_sat_library as (
+
+    select
+        *,
+        row_number() over (partition by library_hk order by load_datetime desc) as rank
```

Comment:
The `load_datetime` is the time the data has been updated in the vault? 
Could there be a case where the time order of the vault differs from the order in the source?
What if a "history" record was changed / added (but it's not the "current" record) in the source?
(not sure it will ever happen, but imagine the source "reloading" or "updating" it's history due to an outage etc)

Reply:
Sharp eye, Flo!

1) The `load_datetime` is the time when warehouse loaded the record. Currently, it is normal satellite table. So, it is append only history table by itself inside DCL vault.
2) I have then transformed via Glue for non-CDC SpreadSheet source into timestamped CDC source at staging PSA layer -- of which, it will "timestamp" the tracking sheet records on daily change summary.
3) So, the DCL vault satellite source is feed by this PSA spreadsheet CDC. And, it detects latest changes upon daily basis.

```
Google sheet source --> Glue (PSA timestamped CDC SpreadSheet table) --> track at vault layer sat_library_description changes
```

Reply:
OK. 
And for the metadata manager, the updates would only come from the "current" data and not from the MM history... 
I guess all good then.
Ta!

Reply:
Correct. 

MM history table is still useful for CDC purpose down the track. Those metadata entries that does not come from Lab tracking sheet, example PDAC-HMF. I will have a plan for them as well. A bit down the line.

Reply:
> segregate research specific meta entries; example PDAC-HMF

Ah, this has been explored in `grimmond_lims` - a research group specific mart meta table...

https://github.com/umccr/orcahouse/blob/main/orcavault/models/mart/grimmond/grimmond_lims.sql#L76-L77

https://github.com/umccr/orcahouse/blob/main/orcavault/models/mart/grimmond/grimmond_lims.sql#L88-L102